report/04_implementation/01_flutter_application.md

```diff
@@ -50,3 +177,10 @@ are clearly separated from each other so that errors or changes only have to
 be carried out once and not several times. In addition, the individual areas
 should be easy to find by other programmers.
 
+### Testing
+
+Testing within soTired was considered as important as every other part of the 
+app. Every function of the config manager and the database manager is 
+rigorously unit tested which ensures a certain amount of code quality.  
```

Comment:
```suggestion
rigorously unit tested which ensures a certain amount of confidence when changing code.  
```

Reply:
I applied both because I think code quality is also ensured with the help of tests 👍🏿